libkatana_python_native/src/PropertyGraph.cpp

```diff
@@ -0,0 +1,742 @@
+#define NPY_NO_DEPRECATED_API NPY_1_11_API_VERSION
```

Comment:
What's this?

Reply:
It tells the numpy native headers what API version I expect. That way it will not issue specific warnings. I cannot exactly remember. https://numpy.org/devdocs/reference/c-api/deprecations.html#deprecation-mechanism-npy-no-deprecated-api